client/README.md

```diff
@@ -0,0 +1,89 @@
+# React + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+
+Currently, two official plugins are available:
+
+- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
+- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+
+# 🛠️ **Setup Instructions for New Contributors**
+
+Welcome to the project! 🎉 Follow these simple steps to set up your local development environment. This will ensure you avoid any errors related to missing configurations or dependencies.
+
+---
+
+### 1️⃣ **Clone the Repository**
+
+Start by cloning the repository to your local machine:
+
+```bash
+git clone https://github.com/yourusername/your-repository.git
+cd your-repository
+```
+
+---
+
+### 2️⃣ **Install Project Dependencies**
+
+Ensure that you have Node.js installed (version 14 or above is recommended). Then, install the required dependencies using npm or Yarn:
+
+- Using npm
+```bash
+npm install
+```
+- Or if you want to use Yarn
+```bash
+yarn install
+```
+
+### 3️⃣ **Generate Missing Configuration Files**
+
+As some configuration files were deleted, you'll need to regenerate or create them manually. Here are the necessary steps for each file:
+## **For TailwindCSS:**
+Regenerate the ```tailwind.config.js``` file:
+```bash
+npx tailwindcss init
+```
+This will generate the ```tailwind.config.js``` file.
+
+## **For Vite:**
```

Comment:
looks like each configuration needs a separate step, in this case it would be better to put these configuration in back in code.

Reply:
You are right, it would.
But we can discuss it in deep on call.